community/communities.md

```diff
@@ -118,4 +118,9 @@ Turkic Interlingua (**TIL**) is “a community of NLP enthusiasts with a special
 
 > Turkic Interlingua (**TIL**) is a community of researchers, Machine Learning engineers, language enthusiasts and community leaders whose mission is to develop language technologies, collect diverse datasets, and explore linguistic phenomena through the lens of academic research for Turkic languages.
 
-[turkic-interlingua.org/](https://turkic-interlingua.org/)
+TIL was founded in September 2020 and currently brings together over 100 NLP enthusiasts together from 10+ countries and regions. 
+
+More info about the community can be found at [turkic-interlingua.org/](https://turkic-interlingua.org/).
```

Comment:
For these 2, it's easier to read if they're just bullet points (and later frontmatter - @cefoo)

> - Website: [turkic-interlingua.org](https://turkic-interlingua.org/)
> - Papers and datasets: [https://github.com/turkic-interlingua/til-mt](https://github.com/turkic-interlingua/til-mt)

I think the bullet points could even be:
> - Website: ...
> - Papers: ...
> - Datasets: ...
> - Models: ...
assuming there are potential deep links to those things.